coala_quickstart/generation/SettingsClass.py

```diff
@@ -354,3 +354,7 @@ def collect_bear_settings(bears):
         for bear in bears[language]:
             bear_settings_obj.append(BearSettings(bear))
     return bear_settings_obj
+
+
+def build_bear_settings(bears):
```

Comment:
not replied to at https://github.com/coala/coala-quickstart/pull/288#discussion_r209432723

Reply:
are you talking about the entire function? I thought it was just related to the `TODO`